StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1600UnitTests.cs

```diff
@@ -188,8 +188,8 @@ public async Task TestFieldWithoutDocumentationAsync()
             await this.TestFieldDeclarationDocumentationAsync(string.Empty, false, false).ConfigureAwait(false);
             await this.TestFieldDeclarationDocumentationAsync("private", false, false).ConfigureAwait(false);
             await this.TestFieldDeclarationDocumentationAsync("protected", true, false).ConfigureAwait(false);
-            await this.TestFieldDeclarationDocumentationAsync("internal", true, false).ConfigureAwait(false);
-            await this.TestFieldDeclarationDocumentationAsync("protected internal", true, false).ConfigureAwait(false);
+            await this.TestFieldDeclarationDocumentationAsync("internal", false, false).ConfigureAwait(false);
+            await this.TestFieldDeclarationDocumentationAsync("protected internal", false, false).ConfigureAwait(false);
```

Comment:
❗️ These should definitely not change. The test is running with `documentInternalElements` set to true. These elements have internal accessibility, so with this configuration a diagnostic is expected. I added tests in #2779 (specifically commit 5712cf82fe851afe2c3e5e0a02ec6f5709861a31) to verify that the analyzer already correctly accounts for `documentInternalElements` set to false.